### DISCUSSION TITLE
Live fuel moisture content

### DISCUSSION THREAD

**Discussion by rosiealice** [NGEET/fates#980] [Papers]
This paper (just out) looks at live fuel moisture content and proposes a new model for its prediction. Might be an interesting candidate for inclusion @adamhb @jkshuman @lmkueppers @ckoven ?

**Comment by jkshuman** [comment]
@rosiealice can you share a link to the paper?

**Reply by rosiealice** [reply]
Haha. Yes. Sorry. I'll go find it again ;)

**Reply by rosiealice** [reply]
https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1365-2435.14271
